### ISSUE TITLE
`@.` macro is incorrectly flagged as missing reference

### ISSUE THREAD

**Issue by piever** [julia-vscode/StaticLint.jl#198]
The `@.` macro is marked as a missing reference in the vs code extension. The `@__dot__` macro on the other hand works fine.

For instance the following code:
```julia
x, y = rand(10), rand(10)
@. x = y
@__dot__ x = y
```
appears as

![dot_missing_ref](https://user-images.githubusercontent.com/6333339/85954015-64fb0380-b96c-11ea-8924-aef352f708cb.png)
